### PR TITLE
Setup Webpack

### PR DESCRIPTION
**Pull request by initseis**
In this branch I:

- Set up Webpack and the loaders

### CODE REVIEW

**Comment by initseis** [comment]
Trying PR

**Comment by sevinchek** [comment]
https://github.com/sevinchek/setup-webpack/blob/7308718f60cd25c58dc78eb575d95de5a7c81569/src/index.js#L1-L13

Whyyy?

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
```
Comment (initseis) [inline .eslintrc.json:5]: Whyyy?

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [ 1, {
+      "js": "always", "json": "always"
+    }]
+  },
+  "ignorePatterns": [
+    "dist/",
```
Comment (initseis) [inline .eslintrc.json:22]: why thiss?

File: .stylelintrc.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": ["stylelint-config-standard"],
+  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": true,
+    "csstree/validator": true
```
Comment (initseis) [inline .stylelintrc.json:7]: hahahaha

**Review by initseis — COMMENTED** [review]
Good job so farrr!

File: .eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
```
Comment (initseis) [inline .eslintrc.json:3]: asasasas

**Pull request merged**